metaworld/envs/pad_obs_dim_env.py

```diff
@@ -0,0 +1,82 @@
+"""Wrapper class that pad zeros."""
+import gym
+import numpy as np
+
+
+class PadObsDimEnv(gym.Wrapper):
```

Comment:
can we just do this in the MultiTaskEnv wrapper?

Reply:
fixed